org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/project/MavenUpdateConfigurationChangeListener.java

```diff
@@ -40,10 +41,10 @@ public class MavenUpdateConfigurationChangeListener implements IResourceChangeLi
 
   @Override
   public void resourceChanged(IResourceChangeEvent event) {
-    if(isDisabled()) {
+    if(isAutoConfigurationUpdateDisabled()) {
```

Comment:
This must be removed or disbaled in the workspace can't be changed by a project.